src/iperf_tcp.c

```diff
@@ -24,8 +24,6 @@
  * This code is distributed under a BSD style license, see the LICENSE
  * file for complete information.
  */
-#include "iperf_config.h"
```

Comment:
Why get rid of this?  I admit the code isn't very consistent in this area, but I tend to lean in the direction of "when in doubt, add the autoconf-generated header", particularly if it's not immediately obvious how that header would get included in the compile (through `src/iperf.h` apparently).  Is this a style change or is there something functional that it fixes?

Reply:
It's for consistency: none of the system include files should be affected by anything in that file (or any files it includes in-tree).  The first in-tree header file that gets included is `"iperf.h"` which promptly includes `"iperf_config.h"`.  This is what `src/iperf_udp.c` does.